src/core/metrics/exporter/send.rs

```diff
@@ -0,0 +1,152 @@
+use log::{debug, trace};
+use std::collections::HashMap;
+use std::io::Write;
+use std::sync::mpsc::Receiver;
```

Comment:
Switch to async mpsc channel

Reply:
I have no idea, how to do it:( 

metrics crate has sync trait with sync functions for recorder (which has functions for sending every single metric) and I need to call from sync method another method of object, which is not sync (async `push_metrics` from sync `increment_counter`). I don't understand how to send do it (can't do method of object async)